src/server/scripts/Kalimdor/CavernsOfTime/BattleForMountHyjal/boss_archimonde.cpp

```diff
@@ -387,18 +398,62 @@ class boss_archimonde : public CreatureScript
                             DoAction(ACTION_ENRAGE);
                     events.ScheduleEvent(EVENT_DISTANCE_CHECK, 5s);
                     break;
+                case EVENT_PROTECTION_OF_ELUNE: // hp below 10% only cast finger of death
+                    events.Reset();
+                    events.ScheduleEvent(EVENT_FINGER_OF_DEATH_LAST_PHASE, 1s);
+                    CastProtectionOfElune(true);
+                    me->GetMotionMaster()->Clear();
+                    me->GetMotionMaster()->MoveIdle();
```

Comment:
there might be something else to call than Clear() + MoveIdle(), I don't remember though what was the right function

Reply:
At last phase.Archimode should not move,only case finger of death.I want to prevent Archimode move.

Reply:
Updated this code.Removed